### ISSUE TITLE
A11y_Visual Studio Code Services_Download_Platform Specific_Keyboard:Keyboard focus is not visible on image link.

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/vscode-docs#5121]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Keyboard; #WCAG2.4.7; #FTP; #ChromiumEdge;

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps:**

1.Launch the application Visual Studio Code Services
2.Navigate to "Download" button.
3.Hit on any of the platform(Windows/Linux/MacOS) links or any installer type(*64 and *32).
4.Observe that keyboard focus is not visible on image link present under visual studio code in service heading.

**Actual:**
Keyboard focus is not visible on image link present under visual studio code in service heading.

**Expected Result:**
keyboard focus should clearly visible on image link present under visual studio code in service heading

**User Impact:**
Users with low vision may not understand the content clearly if keyboard focus is not properly visible.

**Attachment:**

![A11y_Visual Studio Code Services_Download_Platform Specific_Keyboard](https://user-images.githubusercontent.com/97668775/149719894-60379cbe-b996-4fc9-90a8-fc5bc4e4fbe8.gif)

**Comment by gregvanl** [comment]
Duplicate of `no repro` https://github.com/microsoft/vscode-docs/issues/5097

**Comment by poojapatil36** [comment]
Please don't close this bug, the bug is not fixed.

**Comment by gregvanl** [comment]
The bug has been fixed but is not yet published to the live code.visualstudio.com site. It should be published later today

**Comment by poojapatil36** [comment]
Verified the issue, issue is fixed.
please find below attachment.
Github tag: #Closed;
![5121_A11y_Visual Studio Code Services_Download_Platform Specific_Keyboard](https://user-images.githubusercontent.com/97668775/218079568-f6766312-df5a-4b6a-a5fa-456a5a217136.png)
